posthog/api/web_js.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+from django.http import HttpRequest, HttpResponse, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from statshog.defaults.django import statsd
+
+from posthog.logging.timing import timed
+from posthog.plugins.web import get_transpiled_web_source, get_web_config_from_schema
+from posthog.utils import cors_response
+
+
+@csrf_exempt
+@timed("posthog_cloud_web_js_endpoint")
+def get_web_js(request: HttpRequest, id: int, token: str):
+    # handle cors request
+    if request.method == "OPTIONS":
+        return cors_response(request, JsonResponse({"status": 1}))
```

Comment:
Actually, I'll punt... because [unless I'm really off](https://stackoverflow.com/a/71502847), this is not a CORS request at all.

```
<!-- Not a CORS request -->
<script src="https://example.com/script.js"></script>

<!-- CORS request -->
<script type="module" src="https://example.com/script.js"></script>
```

The four horsemen of software development:
- Caching
- Naming Things
- Off-by-one errors
- CORS